feat: rewrite Iterate with JavaScript While Loops challenge (#37697)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-while-loops.english.md

## Instructions
<section id='instructions'>

> Push the numbers 5 through 0, in that order and including both the 5 and 0, to <code>myArray</code> using a <code>while</code> loop.

Reviewer: ```suggestion
Add the numbers 5 through 0 (inclusive) in descending order to <code>myArray</code> using a <code>while</code> loop.
```
